include/LocalHostStats.h

```diff
@@ -43,7 +43,12 @@ class LocalHostStats: public HostStats {
   struct ndpi_hll hll_contacted_hosts;
   double old_hll_value, new_hll_value, hll_delta_value;
   DESCounter contacted_hosts;
-  
+
+  /* Estimate the number of contacted countries using HyperLogLog */
+  struct ndpi_hll hll_countries_contacts;
+  double old_hll_countries_value, new_hll_countries_value, hll_delta_countries_value;
```

Comment:
What's the rationale of using double here? Countries in the world are < 255, so why don't you use u_int8?

Reply:
I am also not sure that using HLL is a good thing here. Again, countries are < 255, did you estimate the memory used by HLL with reference to a plain bitmap for example?

Reply:
we have estimated that the size of a hll counter initialized with 8 bits is 256 bytes, so we have decided to decrease it to 5 or 6 bits in order to occupy 32 or 64 bytes.  We believe that the error generated by the lower precision should not be a problem in estimating the number of countries contacted.  By doing so we occupy the same memory of a 256 bit bitmap which would correspond to 32 bytes occupied

Reply:
Referenced commit with the fix still  has double. What is the rationale of using doubles to keep a number <255?

Reply:
There's no rationale, it was my oversight. I'll fix it right now.